op-e2e/system/conductor/sequencer_failover_setup.go

```diff
@@ -307,6 +307,9 @@ func setupBatcher(t *testing.T, sys *e2esys.System, conductors map[string]*condu
 		DataAvailabilityType:         batcherFlags.CalldataType,
 		ActiveSequencerCheckDuration: 0,
 		CompressionAlgo:              derive.Zlib,
+		ThrottleThreshold:            0, // no DA throttling
+		ThrottleInterval:             2 * time.Second,
```

Comment:
same

Reply:
Changed to make a ThrottleInterval of 0 to mean do-not-start-throttling-loop, and defaulted the threshold flag to a higher non-zero value.